documentation/content/Extra Customization/revealjs.md

```diff
@@ -0,0 +1,43 @@
+---
+author: Pablo Iranzo Gómez
+title: Pelican-Elegant and RevealMD presentations
+tags: pelican, foss, blog, python, github, blog-o-matic, linux, pandoc
+layout: post
```

Comment:
What is this variable for? `layout`.

Reply:
It's in the template I use for my blog, probably from jekyll time :) (removed)